app/templates/companies/new.hbs

```diff
@@ -20,27 +20,24 @@
   <div class="row">
     <div class="{{if isMobileApp 'small-7 large-8' 'small-12'}} columns">
       {{#form-control}}
-        {{input type='text' value=name required='true' class='company-input'}}
+        {{input type='text' value=name required='true' class='company-input' placeholder=(t 'donor_details.company.enter_name')}}
         <div class='input-error'>
-          {{t 'donor_details.company.company_name_validation'}}
+          {{t 'donor_details.company.enter_crm_id'}}
         </div>
       {{/form-control}}
     </div>
   </div>
 
   <div class="row">
     <div class="{{if isMobileApp 'small-7 large-8' 'small-12'}} columns">
-      {{t 'donor_details.company.crm_id'}}
+      {{t 'donor_details.company.crm_id'}} {{t 'donor_details.company.optional'}}
     </div>
   </div>
 
   <div class="row">
     <div class="{{if isMobileApp 'small-7 large-8' 'small-12'}} columns">
       {{#form-control}}
-        {{numeric-input value=crmId id="crm-id" name="crm-id" maxlength="21" required=true class='company-input' pattern="\d{1,8}"}}
-        <div class='input-error'>
-          {{t 'donor_details.company.crm_id_validation'}}
-        </div>
+        {{numeric-input value=crmId id="crm-id" name="crm-id" maxlength="21" class='company-input' pattern="\d{1,8}" placeholder=(t 'donor_details.company.crm_id_validation')}}
```

Comment:
I think you'll need to change placeholder here :)

Reply:
ohh yes, sorry my bad.